bin/s3same

```diff
@@ -1,16 +1,42 @@
 #!/usr/bin/env python
 
 from os.path import abspath, expanduser, exists
+from time import sleep
+from itertools import cycle
+from datetime import datetime, timedelta
 import click
 from dotenv import load_dotenv
-from s3same import artifact_yaml
+from s3same import artifact_yaml, iam_nuke
 
 env_path = abspath(expanduser('~/.s3same'))
 if exists(env_path):
     load_dotenv(env_path)
 
+def nuke(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+    if not click.confirm('Are you sure you want to nuke the whole s3same setup from IAM?'):
+        click.echo('Aborted.')
+        ctx.exit()
+    click.secho(
+            '\nDoing this will make any s3same-generated credentials invalid.\nThere is NO WAY to undo this.\n',
+            bold=True)
+    click.echo('Take a moment to think it over…   ', nl=False)
+    spinner = cycle(['|', '/', '-', '\\'])
```

Comment:
The spinner in particular or the double-confirm with the "No, really, think about it" delay?